packages/@lwc/synthetic-shadow/src/faux-shadow/html-element.ts

```diff
@@ -4,12 +4,8 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import featureFlags from '@lwc/features';
 import { isNull, isFalse, defineProperties, defineProperty } from '@lwc/shared';
 
-import { getInnerText } from '../3rdparty/inner-text';
```

Comment:
should we remove `3rdparty/inner-text.ts`?

Reply:
Missed that! Removed.